koku/masu/database/gcp_report_db_accessor.py

```diff
@@ -134,8 +132,6 @@ def populate_line_item_daily_summary_table_trino(
             "schema": self.schema,
             "table": TRINO_LINE_ITEM_TABLE,
             "source_uuid": source_uuid,
-            "year": invoice_month_date.strftime("%Y"),
-            "month": invoice_month_date.strftime("%m"),
```

Comment:
We still need a year and a month to update here as these are how the data is partitioned.